src/app/components/home/home.component.scss

```diff
@@ -0,0 +1,30 @@
+.background-img {
+	background-image: url('https://as1.ftcdn.net/jpg/02/06/22/86/500_F_206228600_BERXttwnYwOX7UHvuWdTGnf3bc9Flqbn.jpg');
+	background-position: center;
+	background-repeat: no-repeat;
+	background-size: cover;
+	position: fixed;
+	width: 100%;
+	height: 100%;
+	filter: blur(2px);
+}
+
+.logo {
+	background-color: rgba(0,0,0, 0.3);
+	color: white;
+	font-weight: bold;
+	font-family: 'Uncial Antiqua', cursive;
+	text-transform: uppercase;
+	top: 50%;
+	left: 50%;
+	transform: translate(-50%, -50%);
+	z-index: 1;
+	width: 100%;
+	position: absolute;
+  height: 322px;
+	text-align: center;
+	display: flex;
+	justify-content: center;
+	align-items: center;
+	box-shadow: inset 0 0 27px 9px rgba(0,0,0,0.69);
+}
```

Comment:
Please group your css rules. Here is an example: https://github.com/necolas/idiomatic-css